libevmasm/GasMeter.cpp

```diff
@@ -67,11 +68,10 @@ GasMeter::GasConsumption GasMeter::estimateMax(AssemblyItem const& _item, bool _
 		{
 			ExpressionClasses::Id slot = m_state->relativeStackElement(0);
 			ExpressionClasses::Id value = m_state->relativeStackElement(-1);
-			if (classes.knownZero(value) || (
-				m_state->storageContent().count(slot) &&
-				classes.knownNonZero(m_state->storageContent().at(slot))
-			))
-				gas = GasCosts::sstoreResetGas; //@todo take refunds into account
+			if (classes.knownZero(value) ||
+				(m_state->storageContent().count(slot) &&
+				 classes.knownNonZero(m_state->storageContent().at(slot))))
+				gas = GasCosts::sstoreResetGas;	 //@todo take refunds into account
```

Comment:
Yeah, I guess those nitty gritty details are probably still hard to make clang-format obey to...